style.css

```
@@ -480,3 +474,226 @@ img {
     width: 40px;
   }
 }
+
+/* signup */
+.signup {
+  height: 790px;
+  padding: 80px 0px;
+  gap: 30px;
+  background-color: var(--color-accent-yellow);
+}
+
+.signup-container {
+  width: 800px;
+  height: 630px;
+  gap: 40px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: auto;
+}
+
+.signup-section-head {
+  width: 120px;
+  font-family: var(--font-family-heading);
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 48.17px;
```

```suggestion
  line-height: 1.5;
```
line-heightは可能であれば単位なしの値で設定することが望ましいです。
Figma上だと小数点以下の値まで表示されますが、そこまで厳密に実装する必要があるケースは少ないです。
また、小数点以下のpxの扱いはブラウザによって異なるという話もあるため注意が必要です。

参考：
- https://developer.mozilla.org/ja/docs/Web/CSS/line-height
- https://unformedbuilding.com/articles/after-the-decimal-point-in-css/ （10年前の記事なので現在どうなのかは不明）